src/targets/mac/mod.rs

```diff
@@ -0,0 +1,92 @@
+use cocoa::appkit::NSScreen;
+use cocoa::base::{id, nil};
+use cocoa::foundation::NSString;
+use core_graphics_helmer_fork::display::{CGDirectDisplayID, CGMainDisplayID};
+use objc::{msg_send, sel, sel_impl};
+use screencapturekit::{sc_display::SCDisplay, sc_shareable_content::SCShareableContent};
+
+use super::Target;
+
+pub use core_graphics_helmer_fork::display::CGDisplay;
+
+fn get_localized_display_name(display_id: CGDirectDisplayID) -> Option<String> {
+    unsafe {
+        // Get all screens
+        let screens: id = NSScreen::screens(nil);
+        let count: u64 = msg_send![screens, count];
+
+        for i in 0..count {
+            let screen: id = msg_send![screens, objectAtIndex: i];
+            let device_description: id = msg_send![screen, deviceDescription];
+            let display_id_number: id = msg_send![device_description, objectForKey: NSString::alloc(nil).init_str("NSScreenNumber")];
+            let display_id_number: u32 = msg_send![display_id_number, unsignedIntValue];
+
+            if display_id_number == display_id {
+                // Get the localized name
+                let localized_name: id = msg_send![screen, localizedName];
+                let name: *const i8 = msg_send![localized_name, UTF8String];
+                return Some(
+                    std::ffi::CStr::from_ptr(name)
+                        .to_string_lossy()
+                        .into_owned(),
+                );
+            }
+        }
+
+        None
+    }
+}
+
+pub fn get_targets() -> Vec<Target> {
+    let mut targets: Vec<Target> = Vec::new();
+
+    let content = SCShareableContent::current();
+
+    // Add displays to targets
+    for display in content.displays {
+        let id: CGDirectDisplayID = display.display_id;
+        let title = get_localized_display_name(id).unwrap_or_else(|| "Unknown Display".to_string());
```

Comment:
I'm not sure how reliable the `get_localized_display_name` func. 
In the unwrap_or_else, can we return "Unknown Display #x" instead of just unknonwn display. It could be helpful if the user has multiple Unknown Displayss

Reply:
Yes makes sense, updating this.